app/actions/nav.js

```diff
@@ -2,18 +2,6 @@
 
 import * as types from './types';
 
-export function navBack() {
```

Comment:
Removed the planned navigation feature, figuring that the undo functionality will be sufficient, at least for this.